hologram_dashboard/extension/src/app/dashboard/dashboard.component.html

```diff
@@ -0,0 +1,6 @@
+<div class="inline">
+  <button #CHIPPER (click)="onClick(CHIPPER.innerHTML)">Chipper</button>
```

Comment:
I feel we probably should not replace the inner HTML directly. Could you pass the object to the onClick function, and onClick() would construct the dashboard using the object?
I searched and found this (might be a little out-dated) 
https://stackoverflow.com/questions/44264354/display-a-component-after-a-clickevent-angular-4

Reply:
that is the plan, the innerHTML here is only so that I can know which button is pressed.